core/src/components/card/card.tsx

```diff
@@ -131,12 +136,14 @@ export class Card implements ComponentInterface, AnchorInterface, ButtonInterfac
   }
 
   render() {
+    const { shape } = this;
     const theme = getIonTheme(this);
 
     return (
       <Host
         class={createColorClasses(this.color, {
           [theme]: true,
+          [`card-${shape}`]: shape !== undefined,
```

Comment:
We probably want to do something like this to only support the shape in the `"ionic"` theme:

- https://github.com/ionic-team/ionic-framework/blob/7ce115993aa2be4ba70881e5767e4ffd3fedfe21/core/src/components/button/button.tsx#L219-L235
- https://github.com/ionic-team/ionic-framework/blob/7ce115993aa2be4ba70881e5767e4ffd3fedfe21/core/src/components/button/button.tsx#L352
- https://github.com/ionic-team/ionic-framework/blob/7ce115993aa2be4ba70881e5767e4ffd3fedfe21/core/src/components/button/button.tsx#L391

Reply:
Done ✅